scene/3d/skeleton_3d.h

```diff
@@ -223,6 +227,8 @@ class Skeleton3D : public Node3D {
 	Quaternion get_bone_pose_rotation(int p_bone) const;
 	Vector3 get_bone_pose_scale(int p_bone) const;
 
+	void init_bone_poses();
```

Comment:
What is this for and why is it needed? I will insist that for code only needed by the importer, adding code to Skeleton should be avoided.

Reply:
@reduz See https://github.com/godotengine/godot/issues/63465, Since this function was already hard-coded into SkeletonEditor, so implementing it in the Skeleton itself would be useful in cases where you want to reset the animation or posing.

Reply:
Omit for now. We will send a separate PR.

Reply:
Separated as #63486.